tools/src/scie_pants/pants_release_tags.json

```diff
@@ -366,5 +366,286 @@
+  "release_2.13.0.dev2": "238b8a9c6876ed06d8b61b4cde63b35d666d2e35",
+  "release_2.13.0.dev3": "abe8f4555bc3e844e7aec20764bb7cd9e603a365",
+  "release_2.13.0.dev4": "bf013980d556bf4e3768e1db92b95eb5fb5bcc53",
+  "release_2.13.0.dev5": "4ff44d49555044eef9695a93396ed975c17a0bf5",
+  "release_2.13.0a0": "d1dd32a288d620b3bb8cef30b45cae44e2d062a8",
+  "release_2.13.0a1": "9ca295a98677fe9f1d734356ed06ac0023322bd5",
+  "release_2.13.0rc0": "6f458e10dce8d2eec58f54a0fa41f5295535315c",
+  "release_2.13.0rc1": "051c17abf74f38f8f803f43d37cc725058091108",
+  "release_2.13.0rc2": "b0e7422ece51f00f9b97f480987d52153d674132",
+  "release_2.13.0rc3": "6e9061844b942d408f8f0623cc33e1105dc7871d",
+  "release_2.13.1": "7317a67cdf877fcd53f731ceef0d5c3d3734b89a",
+  "release_2.13.1rc0": "c8fab3734f57c87670a28e4fd32f2a44e3e5781e",
+  "release_2.13.1rc1": "a01ec979a3a5bca05b2b5a5d566d816102a86ed3",
+  "release_2.13.1rc2": "441225f1d8f59bc775b0757598e99c9a50063c95",
+  "release_2.14.0": "fae1f69f797f3c6df9a97b47adc7a2290f916743",
+  "release_2.14.0.dev0": "9355d6e3a82544f5d261a958d1be972f15026246",
+  "release_2.14.0.dev1": "54b00e3f6abc9d2fdf7683be5f78d3a5287e4cbb",
+  "release_2.14.0.dev2": "7589c34dac08809ce8fe385d35ab82ceb8059f63",
+  "release_2.14.0.dev3": "f23a0145a8ea7bc207a5101ee7ad4bcc99341bea",
+  "release_2.14.0.dev4": "165bb82841ba83768afcd22f079936813a47030d",
+  "release_2.14.0.dev5": "92d6ebea06cae6c8250dbbca0cb2c66e228065c7",
+  "release_2.14.0.dev6": "370e1e17f24e00901a0c4ee5f8950252c5d44874",
+  "release_2.14.0a0": "9449f42fb37c2663f28da7b33bb4d91d3d1851e7",
+  "release_2.14.0a1": "7270296ab3d08faf8e38bb089caf2f9eca414855",
+  "release_2.14.0rc0": "7eaf2fbb9bfdfb6faf99683375fbccab3db03f13",
+  "release_2.14.0rc1": "c15d5711a9be1e355dbf467e9c98f398546127bc",
+  "release_2.14.0rc2": "e9d0d67d302d2794dd2d9eaa8635b556ba6c4cb9",
+  "release_2.14.0rc3": "0cc05249be406c1210decd892bc3de10bc808e6f",
+  "release_2.14.0rc4": "d5a0bccf66b80ea7508e78fc626d8cbe257a0920",
+  "release_2.14.0rc5": "aa6b0705542437522c06a773d84230df7d26adfc",
+  "release_2.15.0.dev0": "f66cd361d7b06d5eefaa551af99fbeea2a6bc4a5",
+  "release_2.15.0.dev1": "6dda95b425203db32f05eb3d4317655929162332",
+  "release_2.15.0.dev2": "0b76f23e6def63653620d91e2b94611be59b558e",
+  "release_2.15.0.dev3": "a13175ea1d729e0b187fb158e584fc7764533584",
+  "release_2.15.0.dev4": "458c7f04e32cdfba61b2d803d65a077c7d0165fe",
+  "release_2.15.0.dev5": "a21ac88803801cd729e3fb5ff2c187831d708d73",
+  "release_2.15.0a0": "521ee5fe06556233dcaaf1eea4cfe1d564f63642",
+  "release_2.15.0a1": "a7d97014a9ef5626dfa92ab45814ca173f9b60da",
+  "release_2.15.0rc0": "54712e15fdee9ea6cb38e134761222eb3284a3a0",
+  "release_2.16.0.dev0": "c7395e4504738da024dc2514ef4bb9316359ab07",
+  "release_2.16.0.dev1": "57fecb6ce9eee75be20f28fc6c19feb572fa841d",
+  "release_2.16.0.dev2": "c43206feac8cf420c3f3c65fbb9929d75a46f4da",
+  "release_2.2.0": "81fd0314465d9d1607bd3bc9b36d608434236e75",
```

Comment:
Could you remind me if more releases of 2.16 and 2.17 (things before `2.18.0.dev5`) need to appear here?

Reply:
This file is just a cache (or whatever the baked-in, static version of a cache is) which is checked first before making an unauthenticated GitHub request.

Reply:
Honestly, I feel like if the rate limit errors were material we'd see them by now. I'm much more worried about getting the SHA wrong. 

If we really wanted to, is suggest we did that in a dedicated PR